rllib/connectors/common/add_states_from_episodes_to_batch.py

```diff
@@ -199,16 +199,24 @@ def __call__(
         # Also, let module-to-env pipeline know that we had added a single timestep
         # time rank to the data (to remove it again).
         if not self._as_learner_connector:
-            data = tree.map_structure(lambda s: np.expand_dims(s, axis=0), data)
+            data = tree.map_structure(
+                # Expand on axis 0 (the to-be-time-dim) if item has not been batched'
+                # yet, otherwise axis=1 (the time-dim).
+                lambda s: np.expand_dims(
```

Comment:
`expand_dims` has an impact on the memory management of numpy arrays. Is here maybe a 'reshape'  action possible or do we need to fill the new axis with new values? For example do we make a `(32, 4)` array a `(32, 1, 4)` one or a `(32, max_seq_len, 4)` one?

Reply:
Ah, great point! I didn't know that. Maybe we can just replace by a reshape. No we don't expect nor want this axis to be >1. It's the "simple" time-axis=1 add for action computing forward passes.